es/es-server/src/main/java/org/elasticsearch/repositories/Repository.java

```diff
@@ -132,8 +132,15 @@ default Repository create(RepositoryMetaData metaData, Function<String, Reposito
      * @param includeGlobalState include cluster global state
      * @return snapshot description
      */
-    SnapshotInfo finalizeSnapshot(SnapshotId snapshotId, List<IndexId> indices, long startTime, String failure, int totalShards,
-                                  List<SnapshotShardFailure> shardFailures, long repositoryStateId, boolean includeGlobalState);
+    SnapshotInfo finalizeSnapshot(SnapshotId snapshotId,
```

Comment:
Should we also backport the deprecation marker addition to `initializeSnapshot` that was added upstream?

Reply:
Good point, you are right.